paddle/fluid/inference/api/analysis_predictor.h

```diff
@@ -124,8 +137,14 @@ class AnalysisPredictor : public PaddlePredictor {
   FRIEND_TEST(AnalysisPredictor, analysis_off);
   FRIEND_TEST(AnalysisPredictor, analysis_on);
   FRIEND_TEST(AnalysisPredictor, with_gpu);
+
+  friend class QuantizerTest;
```

Comment:
why put `QuantizerTest` in `analysis_predictor.h`?

Reply:
`friend class QuantizerTest;` is in both `AnalysisPredictor` and `AnalysisPredictor::Quantizer` because the `Quantizer` is in `private` section of `AnalysisPredictor`.